lib/ui/widgets/posts_list_item/index.dart

```diff
@@ -16,18 +16,26 @@ import 'widgets/export.dart';
 ///    for such post
 class PostListItem extends StatelessWidget {
   final Post post;
-  PostListItem({Key key, @required this.post}) : super(key: key);
+  final bool displayErrorPost;
+  PostListItem({
+    Key key,
+    @required this.post,
+    this.displayErrorPost = false,
+  }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     Color color = Theme.of(context).cardColor;
-
     if (post.status.value != PostStatusValue.TX_SUCCESSFULL) {
       color = color.withOpacity(0.5);
     }
 
     if (post.status.value == PostStatusValue.ERRORED) {
-      color = Theme.of(context).errorColor.withOpacity(0.75);
+      if (displayErrorPost) {
```

Comment:
Since `displayErrorPost` is only defined in the constructor and never used, I think we can remove it entirely